### PR TITLE
<MarketingLayout/> - add alignItems prop

### PR DESCRIPTION
**Pull request by yogevitz**
### 🔦 Summary

add `alignItems` prop to `<MarketingLayout/>` component

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @milkyfruit 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6275

File: src/MarketingLayout/MarketingLayout.js [hunk 1]
```diff
@@ -58,6 +60,7 @@ class MarketingLayout extends React.PureComponent {
 
   static defaultProps = {
     size: SIZES.small,
+    alignItems: ALIGNMENTS.center,
```
Comment (EgoziE) [inline src/MarketingLayout/MarketingLayout.js:63]: When it comes to `static propTypes` and `static defaultProps`, we must use explicit values for the AutoDocs parser.

```suggestion
    alignItems: 'center',
```

**Review by EgoziE — COMMENTED** [review]
@milkyfruit Ready for UX review

**Review by milkyfruit — APPROVED** [review]
@EgoziE looks great!

**Review by EgoziE — APPROVED** [review]

**Pull request merged**